scripts/grab_reference_docs.sh

```diff
@@ -39,12 +41,37 @@ locate_file() {
     cp ${FILENAME} _docs/${PP}/${FN}
 }
 
-find $WORKDIR/api -type f -name '*.pb.html' | while read line; do
+# Given the path and name to an Istio command, builds the command and then
+# runs it to extract its command-line docs
+#
+# TODO: Even though this CDs into the source tree we've extracted, it's not actually
+# using that as input sources since imports are resolved through $GOPATH and such.
+# I'm not clear what voodoo is needed so I'm leaving this as-is for the time being
+get_command_doc() {
+    COMMAND_PATH=$1
+    COMMAND=$2
+
+    cd $COMMAND_PATH
```

Comment:
pushd/popd would be better so we can exit this function back in the directory it was invoked from.

Not blocking.